src/common/common.py

```
@@ -75,6 +80,11 @@ def save_params(params: dict[str, Any]) -> None:
     Returns:
         dict[str, Any]: Updated parameters.
     """
+    
+    # Check if the workspace is enabled and if a 'params.json' file exists in the workspace directory
+    if not st.session_state.settings["is_workspace_enabled"] == True:
```

Ruff (0.8.2)

Use `st.session_state.settings["is_workspace_enabled"] != True` instead of `not st.session_state.settings["is_workspace_enabled"] == True`

Replace with `!=` operator

(SIM201)